libsql/src/sync.rs

```diff
@@ -0,0 +1,18 @@
+const DEFAULT_MAX_RETRIES: usize = 5;
+pub struct SyncContext {
+    pub sync_url: String,
+    pub auth_token: Option<String>,
+    pub max_retries: usize,
+    pub durable_frame_num: u32,
```

Comment:
We should provide getters for these rather than expose them. I can do this in a follow up.